common/source/docs/common-ie24-fuelcell.rst

```diff
@@ -0,0 +1,204 @@
+.. _common-ie24-fuelcell:
+
+==================================
+Intelligent Energy Fuel Cell 2.4kW
+==================================
+
+.. figure:: ../../../images/ie2400W_fuelcell.jpg
+   :target: ../_images/ie2400W_fuelcell.jpg
+
+
+This page explains how to connect Intelligent Energy's 2.4 kW fuel cell to ArduPilot.  Telemetry from the fuel cell is transmitted via serial to ArduPilot.  This gives the following feedback:
+
+- Hydrogen tank level as a percentage.
+- Battery voltage.
+- Battery current.
+- Battery remaining.
+- Fuel cell state.
+- Error codes.
+- Total power out (log only).
+- Stack Power Module (SPM) power (log only).
+
+
+Setup
+-----
+
+Enable the fuel cell driver by setting :ref:`GEN_TYPE <GEN_TYPE>` to 2.
```

Comment:
I think it would be better to just provide a bullet point list of parameter and what they should be set to like we've done on the richen power page. https://ardupilot.org/copter/docs/common-richenpower-generator.html

Sorry to be annoying but the reader just wants to get in and out as quickly as possible so let's not hide the parameters that need changing within long sentences.

Reply:
I disagree.  Why give users less information?  I acknowledge a balance has to be struck so as to not make the page overly verbose.  However, if sufficient information is provided then users will be better equipped to work out the specifics of how to configure their setup, if it is different from the example given.  Given the way that the fuel cells interact with the battery monitor library is a little odd/unconventional, I think the information provided is needed.  Plus, all parameter names are highlighted in blue so can be easily picked out if users don't want to read all of the info.